src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,58 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+const Tabs = ({ links, listClassName, navClassName }) => {
+  return (
+    <nav className={classNames("p-tabs", navClassName)}>
+      <ul className={classNames("p-tabs__list", listClassName)}>
+        {links.map((link) => {
+          const {
+            active,
+            className,
+            component,
+            label,
+            listItemClassname,
+            ...rest
+          } = link;
+          const Component = component || "a";
+          return (
+            <li
+              className={classNames("p-tabs__item", link.listItemClassName)}
+              key={link.label}
+            >
+              <Component
+                aria-selected={link.active}
+                className={classNames("p-tabs__link", link.className)}
+                data-test={`tab-link-${link.label}`}
+                {...rest}
+              >
+                {link.label}
+              </Component>
+            </li>
+          );
+        })}
+      </ul>
+    </nav>
+  );
+};
+
+Tabs.propTypes = {
```

Comment:
It'd be good to document these props, kinda like https://github.com/canonical-web-and-design/react-components/blob/master/src/components/Accordion/Accordion.js#L49

Reply:
Done!